solr/bin/solr

```diff
@@ -937,8 +814,8 @@ function stop_solr() {
 
 if [ $# -eq 1 ]; then
   case $1 in
-    -help|-h)
-        print_usage ""
+    -help|-h)    
+        run_tool ""    
```

Comment:
[Q] Does nothing in `tidy`/`check` complain about trailing whitespace?  I don't care much, but I'm surprised given how we've leaned into code-formatting in recent years.

Reply:
Not that I know of...   Maybe because this is shell?  I didn't run shellcheck after making changes...